### PR TITLE
i18n-ize meteor email templates

### PR DESCRIPTION
**Pull request by floatinghotpot**

### CODE REVIEW

File: config/accounts.js [hunk 1]
```diff
@@ -51,4 +51,13 @@ if (Meteor.isServer) {
   if (process.env.MAIL_FROM) {
     Accounts.emailTemplates.from = process.env.MAIL_FROM;
   }
+
+  ['resetPassword-subject', 'resetPassword-text', 'verifyEmail-subject', 'verifyEmail-text', 'enrollAccount-subject', 'enrollAccount-text'].forEach((str) => {
+    const words = str.split('-');
+    Accounts.emailTemplates[words[0]][words[1]] = (user, url) => {
+      const lang = (user.profile && user.profile.language) ? user.profile.language : 'en';
+      const name = (user.profile && user.profile.name) ? user.profile.name : user.username;
```
Comment (mquandalle) [inline config/accounts.js]: Could hide the implementation details of the retrieval by creating the `getLanguage()` and `getName()` method on the user object (ie to the helper list defined in `models/users`)?

File: config/accounts.js [hunk 2]
```diff
@@ -51,4 +51,13 @@ if (Meteor.isServer) {
...
+      return TAPi18n.__(`email-${str}`, { user: name, url }, lang);
```
Comment (mquandalle) [inline config/accounts.js]: Doesn’t this require the language code conversion that you implemented in #411?

**Comment by mquandalle** [comment]
:+1: This is an excellent idea, thank you :)

File: config/accounts.js (hunk 2 above)
Comment (floatinghotpot) [inline config/accounts.js]: That code conversion is only for T9n, to set language for meteor account UI resources. Here is for the i18n resources inside WeKan so conversion is not needed.

Comment (mquandalle) [inline config/accounts.js]: Perfect!

**Pull request merged**